src/main/java/frc/robot/subsystems/CoralScorer.java

```diff
@@ -77,6 +77,12 @@ public Command stopCommand() {
     return runOnce(() -> scorerMotor.stopMotor());
   }
 
+  public Command alignCommand() {
+    return intakeCommand()
+        .until(hasCoral().negate())
+        .andThen(reverseCommand().until(hasCoral()).withTimeout(2));
```

Comment:
This Time out is for sure going to be too long. Lets start with something closer to 0.5.